ocp_utilities/must_gather.py

```diff
@@ -33,7 +35,8 @@ def run_must_gather(
 
             Note: flag is optional parameter for must-gather. When it is not passed "--default" flag is used by
             must-gather. However, flag_names can not be passed without script_name
-
+        since (str, optional): since when the data should be collected. format is: '(+|-)[0-9]+(s|m|h|d)'
+        timeout (str, optional): runs the debug pods for specified duration
```

Comment:
in which format? please give an example

Reply:
Could not find any doc specifying it clearly, between various searches, it might be same as --since. https://github.com/search?q=repo%3Aopenshift%2Fmust-gather%20timeout&type=code
I will add an example for seconds.